lib/onfido/models/check_response.rb

```diff
@@ -75,7 +78,9 @@ def self.attribute_map
         :'form_uri' => :'form_uri',
```

Comment:
I dont understand the utility of this hash 😆

Reply:
That auto-generated code come from [this](https://github.com/OpenAPITools/openapi-generator/blob/v7.6.0/modules/openapi-generator/src/main/resources/ruby-client/partial_model_generic.mustache#L36-L43) template from the generator.

Cannot follow that across all the files, but it might be that this hash is built using [this function](https://github.com/OpenAPITools/openapi-generator/blob/v7.6.0/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractRubyCodegen.java#L221-L230) (and below) to remove characters that can be used in json but not as a ruby variable names.

Most of the cases should be the same value, but in some particular case it wouldn't.